CONTRIBUTING.md

```diff
@@ -8,4 +8,4 @@ later releases and we’ll keep you posted on this). However, we are taking issu
 See the [GDK for Unreal C++ coding standards guide]({{urlRoot}}/contributions/unreal-gdk-coding-standards).
 
 ## Warning
-This [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release of the SpatialOS GDK for Unreal is for evaluation and feedback purposes only, with limited documentation - see the guidance on [Recommended use]({{urlRoot}}/index#recommended-use)
\ No newline at end of file
+This is an [alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release of the SpatialOS Unreal GDK, pending stability and performance improvements. The API may change as we learn from feedback - see the guidance on Recommended use, below.
```

Comment:
```suggestion
This is an [alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release of the SpatialOS GDK for Unreal, pending stability and performance improvements. The API may change as we learn from feedback - see the guidance on [Recommended use]({{urlRoot}}/readme/recommended-use).
```

There's no "recommend use" section below on this page, so I tried adding a link to the one in the README.md instead. Not sure if the link is right though.